.ado/apple-pr.yml

```diff
@@ -10,7 +10,8 @@ pr:
   branches:
     include:
       - master
-      - 0.59-stable
+      - 0.62-stable
```

Comment:
Seemed odd this was still on 0.59. @acoates-ms let me know if there's a reason we wouldn't want to update this part like so